### PR TITLE
Document better how to contribute

### LINKED ISSUE #249: Instructions for adding templates
Hi, thank you for this very nice package!

There is a template or two that I would like to add, are there any instructions on how to add new templates to the package?

Thanks!

### PR DESCRIPTION
**Pull request by cderv**
closes #249 

I think it does no harm to add more Getting started information in the README. 

In doubt, here is a PR instead of pushing to master directly.

### CODE REVIEW

**Comment by cderv** [comment]
In fact, let's not bother you with that I just merged and you'll amend later if you think something is missing or not correctly said.

File: README.md [hunk 1]
```diff
@@ -80,3 +83,32 @@ To use **rticles** from RStudio:
     rmarkdown::draft("MyJSSArticle.Rmd", template = "jss_article", package = "rticles")
     rmarkdown::draft("MyRJournalArticle", template = "rjournal_article", package = "rticles")
     ```
+
+## How to contribute ?
```
Comment (yihui) [inline README.md:87]:
```suggestion
## How to contribute? 
```

File: README.md [hunk 2]
```diff
@@ -17,7 +17,10 @@ remotes::install_github("rstudio/rticles")
 
 ## Overview
 
-The **rticles** package provides a suite of custom [R Markdown](http://rmarkdown.rstudio.com) LaTeX formats and templates for various formats, including:
+The **rticles** package provides a suite of custom [R Markdown](http://rmarkdown.rstudio.com) LaTeX formats and templates for various formats. Most of the templates are provided and maintained by the community, and anyone can contribute a new template. See [How to contribute](#how-to-contribute-)
```
Comment (yihui) [inline README.md:20]:
```suggestion
The **rticles** package provides a suite of custom [R Markdown](http://rmarkdown.rstudio.com) LaTeX formats and templates for various formats. Most of the templates are provided and maintained by the community, and anyone can contribute a new template. See [How to contribute](#how-to-contribute) below.
```

File: README.md [hunk 3]
```diff
@@ -17,7 +17,10 @@ remotes::install_github("rstudio/rticles")
...
+
+Including templates and contributors are:
```
Comment (yihui) [inline README.md:22]:
```suggestion
Currently included templates and their contributors are the following:
```

File: README.md [hunk 4]
```diff
@@ -80,3 +83,32 @@ To use **rticles** from RStudio:
     rmarkdown::draft("MyJSSArticle.Rmd", template = "jss_article", package = "rticles")
     rmarkdown::draft("MyRJournalArticle", template = "rjournal_article", package = "rticles")
     ```
+
+## How to contribute ? 
+
+Most of the templates are contributed directly by the users in the community. 
+If you want `rticles` to offer a new journal format, you can contribute by the following way.
```
Comment (yihui) [inline README.md:90]: 1. Github turns a line break in Markdown to `<br/>` (unlike Pandoc, which ignores line breaks by default), which we don't want.
1. We use bold text for **package** names.

```suggestion
Most of the templates are contributed directly by the users in the community. If you want **rticles** to offer a new journal format, you can contribute by the following way.
```

Similar issues in the rest of text below.

**Pull request merged**